# ¿Cuánto vale tu trabajo?
## Salario Mínimo, Desigualdad y Poder Adquisitivo en la Unión Europea

---
## 1. Descripción

Este proyecto analiza la evolución del **salario mínimo en los países de la Unión Europea** desde una perspectiva social y económica: qué puedes comprar realmente con él, cuánto te queda después de pagar el alquiler, cómo afecta a la desigualdad de género, y si tener trabajo es suficiente para no ser pobre.

La pregunta central es directa y muy actual: **¿sirve de algo el salario mínimo en Europa hoy?**

El hilo conductor del proyecto es una paradoja que los datos europeos evidencian con claridad:

> *El salario mínimo ha subido en casi todos los países de la UE durante la última década. Sin embargo, el poder adquisitivo real de los trabajadores con menor renta no ha mejorado al mismo ritmo — porque el coste de vida, especialmente la vivienda, ha crecido mucho más rápido que los salarios.*

Subir el número no es suficiente si todo lo demás sube más. Eso es lo que este análisis intenta demostrar con datos.

---
## 2. Hipótesis

Las hipótesis se plantean **antes de ver los datos** — eso es fundamental en un EDA riguroso. Al final del proyecto comprobaremos cuáles se confirman y cuáles no.

---

**H1 — El salario mínimo nominal engaña** *(hipótesis principal)*  
Los países con mayor salario mínimo en euros no son necesariamente los que ofrecen mayor poder adquisitivo real. Al ajustar por el coste de vida mediante PPS (Purchasing Power Standard), las diferencias entre países se reducen drásticamente.

---

**H2 — El salario mínimo es una herramienta de política feminista**  
Los países con un salario mínimo más alto como porcentaje del salario medio tienen una brecha salarial de género menor. El sistema ha colocado históricamente a las mujeres en los empleos peor pagados. Subir el suelo salarial corrige estructuralmente esa injusticia.

---

**H3 — En la mayoría de la UE, cobrar el mínimo significa no poder pagar el alquiler**  
El salario mínimo no es suficiente para vivir de forma independiente: en la mayoría de países europeos implica destinar más del 40% del sueldo al alquiler, haciendo imposible la emancipación para millones de jóvenes.

---

**H4 — Working poor: tener trabajo no te salva de la pobreza**  
Los países con un salario mínimo más bajo en términos reales concentran mayor proporción de trabajadores en riesgo de exclusión social. El empleo sin salario digno no cumple su función protectora.

---

**H5 — La Generación Z es la primera que trabaja más para vivir peor que sus padres**  
Los salarios mínimos han subido en casi todos los países de la UE, pero la combinación de alquileres disparados y estancamiento salarial real hace que la generación Z tenga peor capacidad adquisitiva que sus padres a la misma edad.

---
## 3. Datasets y fuentes de datos

Todos los datos provienen de **Eurostat**, la oficina estadística oficial de la Unión Europea. Es la fuente más fiable para análisis económicos comparados entre países europeos, ya que estandariza la metodología en todos los estados miembros.

Se utilizan 5 datasets que posteriormente se combinarán mediante **merge** para construir un DataFrame unificado con más de 1.000 filas y más de 10 columnas — los mínimos requeridos para un análisis exploratorio robusto.

| # | Dataset | Código Eurostat | Variables clave | Hipótesis |
|---|---|---|---|---|
| 1 | Salario mínimo mensual | `earn_mw_cur` | €/mes por país y semestre | H1, H2, H3, H4, H5 |
| 2 | PIB per cápita en PPS | `tec00114` | Riqueza ajustada por poder adquisitivo | H1 |
| 3 | Índice de precios de vivienda | `prc_hpi_a` | Evolución precios alquiler y compra | H3, H5 |
| 4 | Índice de desigualdad Gini | `ilc_di12` | Distribución de la renta | H4 |
| 5 | Brecha salarial de género | `sdg_05_20` | Diferencia % salario hombre/mujer | H2 |

**Nota sobre el salario mínimo:** El archivo descargado de Eurostat contiene solo los 10 períodos más recientes. Para el análisis completo se utilizará la **API de Eurostat** — una conexión directa desde Python que permite obtener el histórico completo desde 1999 sin pasar por la interfaz web.

---
## 4. Carga y primera vista de los datos

In [1]:
import pandas as pd
import numpy as np

print('Librerías importadas correctamente')

Librerías importadas correctamente


### Dataset 1 — Salario mínimo mensual (`earn_mw_cur`)

In [2]:
fuente_1 = 'Eurostat - Monthly minimum wages bi-annual data (earn_mw_cur)'

df_1 = pd.read_csv('./data/salario_minimo.tsv', sep='\t')

print(f'Fuente: {fuente_1}')
print(f'Dimensiones: {df_1.shape[0]} filas x {df_1.shape[1]} columnas')
df_1.head()

Fuente: Eurostat - Monthly minimum wages bi-annual data (earn_mw_cur)
Dimensiones: 38 filas x 11 columnas


,"freq,currency,geo\TIME_PERIOD",2021-S2,2022-S1,2022-S2,2023-S1,2023-S2,2024-S1,2024-S2,2025-S1,2025-S2,2026-S1
0,"S,EUR,AL",245,248,269,298,376,385,399,408,408,517
1,"S,EUR,AT",: m,: m,: m,: m,: m,: m,: m,: m,: m,: m
2,"S,EUR,BE",1626,1658,1842,1955,1955,1994,2070,2070,2112,2112
3,"S,EUR,BG",332,332,363,399,399,477,477,551,551,620
4,"S,EUR,CH",: m,: m,: m,: m,: m,: m,: m,: m,: m,: m


### Dataset 2 — PIB per cápita en PPS (`tec00114`)

In [3]:
fuente_2 = 'Eurostat - GDP per capita in PPS (tec00114)'

df_2 = pd.read_csv('./data/pib_percapita.tsv', sep='\t')

print(f'Fuente: {fuente_2}')
print(f'Dimensiones: {df_2.shape[0]} filas x {df_2.shape[1]} columnas')
df_2.head()

Fuente: Eurostat - GDP per capita in PPS (tec00114)
Dimensiones: 42 filas x 13 columnas


,"freq,indic_ppp,ppp_cat18,geo\TIME_PERIOD",2013,2014,2015,2016,2017,2018,2019,2020,2021,2022,2023,2024
0,"A,VI_PPS_EU27_2020_HAB,GDP,AL",29,30,30,30,30,30,30,30,31,33,38,42 p
1,"A,VI_PPS_EU27_2020_HAB,GDP,AT",131,129,128,128,125,126,124,123,121,123,121,119
2,"A,VI_PPS_EU27_2020_HAB,GDP,BA",30,30,31,31,31,32,32,33,33,34,35,35
3,"A,VI_PPS_EU27_2020_HAB,GDP,BE",121,121,119,118,117,116,117,118,117,118,118,117 p
4,"A,VI_PPS_EU27_2020_HAB,GDP,BG",46,48,49,51,52,53,55,57,60,62,63,66


### Dataset 3 — Índice de precios de vivienda (`prc_hpi_a`)

In [4]:
fuente_3 = 'Eurostat - House price index (prc_hpi_a)'

df_3 = pd.read_csv('./data/precios_vivienda.tsv', sep='\t')

print(f'Fuente: {fuente_3}')
print(f'Dimensiones: {df_3.shape[0]} filas x {df_3.shape[1]} columnas')
df_3.head()

Fuente: Eurostat - House price index (prc_hpi_a)
Dimensiones: 330 filas x 11 columnas


,"freq,purchase,unit,geo\TIME_PERIOD",2015,2016,2017,2018,2019,2020,2021,2022,2023,2024
0,"A,DW_EXST,I10_A_AVG,AT",128.63,136.27,144.48,153.57,162.82,175.30,198.25,221.28,212.27,208.89
1,"A,DW_EXST,I10_A_AVG,BE",108.42,110.71,114.46,117.96,122.63,127.83,136.53,144.12,147.53,151.32
2,"A,DW_EXST,I10_A_AVG,BG",92.29 b,99.07,108.70,116.35,121.93,129.22,141.15,163.17 b,180.44,208.82
3,"A,DW_EXST,I10_A_AVG,CY",91.93 b,91.65,91.62,90.26,91.70,94.14,95.32,100.16,107.65,115.22
4,"A,DW_EXST,I10_A_AVG,CZ",105.40,113.10,126.00,136.30,148.90,161.20,193.50,225.50,221.70,232.80


### Dataset 4 — Índice de desigualdad Gini (`ilc_di12`)

In [5]:
fuente_4 = 'Eurostat - Gini coefficient of equivalised disposable income (ilc_di12)'

df_4 = pd.read_csv('./data/gini.tsv', sep='\t')

print(f'Fuente: {fuente_4}')
print(f'Dimensiones: {df_4.shape[0]} filas x {df_4.shape[1]} columnas')
df_4.head()

Fuente: Eurostat - Gini coefficient of equivalised disposable income (ilc_di12)
Dimensiones: 78 filas x 11 columnas


,"freq,age,statinfo,geo\TIME_PERIOD",2016,2017,2018,2019,2020,2021,2022,2023,2024,2025
0,"A,TOTAL,GINI_HND,AL",:,36.8,35.4,34.3,33.2,33.0,31.0,30.2,:,:
1,"A,TOTAL,GINI_HND,AT",27.2,27.9,26.8,27.5,27.0,26.7,27.8,28.1,28.4,:
2,"A,TOTAL,GINI_HND,BE",26.3,26.1,25.7,25.1 b,25.3,24.1,24.7,24.3,24.6,23.4
3,"A,TOTAL,GINI_HND,BG",37.7 b,40.2,39.6,40.8,40.0,39.7,38.4,37.2,38.4,37.7
4,"A,TOTAL,GINI_HND,CH",29.4,30.1,29.7,30.6,31.2,31.4,31.0,31.5,31.0,:


### Dataset 5 — Brecha salarial de género (`sdg_05_20`)

In [6]:
fuente_5 = 'Eurostat - Gender pay gap in unadjusted form (sdg_05_20)'

df_5 = pd.read_csv('./data/brecha_genero.tsv', sep='\t')

print(f'Fuente: {fuente_5}')
print(f'Dimensiones: {df_5.shape[0]} filas x {df_5.shape[1]} columnas')
df_5.head()

Fuente: Eurostat - Gender pay gap in unadjusted form (sdg_05_20)
Dimensiones: 40 filas x 21 columnas


,"freq,unit,nace_r2,geo\TIME_PERIOD",2002,2006,2007,2008,2009,2010,2011,2012,2013,...,2015,2016,2017,2018,2019,2020,2021,2022,2023,2024
0,"A,PC,B-S_X_O,AL",:,:,:,:,:,:,:,:,:,...,:,:,:,6.8,:,:,:,:,:,:
1,"A,PC,B-S_X_O,AT",:,25.5,25.5,25.1,24.3,24.0,23.5,22.9,22.3,...,21.8,20.8,20.7,20.4,20.0,19.2,19.1,18.7,18.3 p,17.6 p
2,"A,PC,B-S_X_O,BE",:,9.5,10.1,10.2,10.1,10.2,9.4,8.3,7.5,...,6.4,6.0,5.8,5.8,4.7,3.2,1.8,0.7,0.7 p,0.7 p
3,"A,PC,B-S_X_O,BG",18.9,12.4,12.1,12.3,13.3,13.0,13.2,15.1,14.1,...,15.5,14.6,14.3,13.9,14.0,12.5,11.9,12.6,13.5 p,12 p
4,"A,PC,B-S_X_O,CH",:,18.6,:,18.4 d,18.4 d,17.8,17.6,17.4,17.6,...,17.9,17.4,17.6,18.6,18.4,18.0,17.0,17.0,17.2 p,16 p


---
## 5. Plan de análisis y justificación del merge

Cada dataset por separado tiene entre 38 y 330 filas. Para alcanzar el mínimo de **1.000 filas y 10 columnas** necesario para un análisis exploratorio robusto, se realizará un **merge** de los 5 datasets en uno único.

Un merge es la operación de combinar dos o más tablas usando una columna común como clave. En este caso la clave será el **país** y el **año** — cada fila del DataFrame final representará un país en un año concreto, con todas sus variables asociadas.

El DataFrame final tendrá esta estructura aproximada:

| pais | anio | salario_minimo_eur | pib_pps | indice_vivienda | gini | brecha_genero |
|---|---|---|---|---|---|---|
| Spain | 2020 | 950 | 91 | 112.3 | 33.0 | 11.9 |
| Germany | 2020 | 1584 | 118 | 108.7 | 31.5 | 18.3 |
| ... | ... | ... | ... | ... | ... | ... |

El EDA completo seguirá esta estructura:

1. **Limpieza de datos** — formato de columnas, valores nulos, tipos de datos
2. **Merge de los 5 datasets** — construcción del DataFrame unificado
3. **Análisis univariante** — distribución de cada variable por separado
4. **Análisis bivariante** — relaciones entre pares de variables
5. **Análisis multivariante** — correlaciones, heatmaps, pairplots
6. **Contraste de hipótesis** — ¿se confirman las 5 hipótesis?
7. **Conclusiones (y plan de acción)**

---
*Datos: Eurostat (ec.europa.eu/eurostat) | Proyecto EDA — Data Science Bootcamp 2026*